specutils/analysis/flux.py

```diff
@@ -178,7 +177,7 @@ def is_continuum_below_threshold(spectrum, threshold=0.01):
     # we are going to calculate based on the S/N if the uncertainty
     # exists.
     if uncertainty and uncertainty.uncertainty_type != 'std':
-        return np.median(flux / uncertainty.quantity) < threshold
+        return np.median(flux / uncertainty) < threshold
```

Comment:
```suggestion
        return np.median(flux / uncertainty.quantity) < threshold
```

We certainly want this with `.quantity` -- `NDUncertainty` classes do not support arithmetic otherwise.